src/main/kotlin/io/snyk/plugin/snykcode/SnykCodeBulkFileListener.kt

```diff
@@ -105,10 +107,9 @@ class SnykCodeBulkFileListener : SnykBulkFileListener() {
         val cache = getSnykCachedResults(project)?.currentSnykCodeResultsLS ?: return
         filesAffected.forEach {
             cache.remove(it)
-            it.virtualFile.getPsiFile(project)?.let { psiFile ->
-                DaemonCodeAnalyzer.getInstance(project).restart(psiFile)
-            }
```

Comment:
We need to remove its `import` and it seems that we didn't need `refreshAnnotationsForOpenFiles` in the end, could it be?